src/hypofuzz/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Adaptive fuzzing for property-based tests using Hypothesis."""
 
-__version__ = "24.9.1"
+__version__ = "24.11.1"

```

Comment:
alas, 😅

Reply:
I have definitely been spoiled by never having to worry about bumping in hypothesis...